webapp/content/js/ext/resources/css/ext-all.css

```diff
@@ -2988,7 +2988,7 @@ input.x-tree-node-cb {
 	-khtml-user-select:none;
 	-moz-user-select:none;
     -webkit-user-select:ignore;
-    -kthml-user-focus:normal;
+    -khtml-user-focus:normal;
```

Comment:
khtml is a rendering engine...

This might be a third party file that shouldn't be changed.

I can eventually send PRs to upstreams, but my queue is quite long...